RHEL/7/input/profiles/ospp-rhel7-server.xml

```diff
@@ -0,0 +1,135 @@
+<Profile id="ospp-rhel7-server" extends="common">
+<title override="true">Protection Profile for General Purpose Operating Systems Version 4.0 - Red Hat Enterprise Linux 7 Server</title>
```

Comment:
This should be a short title, something that can fit into Anaconda profile selection. Perhaps:
- `````NIAP Operating System Baseline`````
- `````NIAP OSPPv4`````

Because the content including this profile would be for RHEL7, there's no need to call that out in the profile title itself.